arroyo/processing/strategies/collect.py

```diff
@@ -14,14 +15,12 @@ class Batch(Generic[TPayload]):
     def __init__(
         self,
         step: ProcessingStep[TPayload],
-        commit_function: Callable[[Mapping[Partition, Position]], None],
     ) -> None:
         self.__step = step
-        self.__commit_function = commit_function
 
         self.__created = time.time()
         self.__length = 0
-        self.__offsets: MutableMapping[Partition, Position] = {}
```

Comment:
I don't think this variable is needed at all anymore, no?

Reply:
Changed to non private. The strategy uses `batch.offsets` as the offsets that get committed.